tests/datavolume_test.go

```diff
@@ -361,4 +361,35 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 			}, timeout, pollingInterval).Should(BeTrue())
 		})
 	})
+
+	Describe("Restarts reporting on import datavolume", func() {
+		It("Should report restarts on failed import", func() {
+			dataVolume := utils.NewDataVolumeWithHTTPImport(dataVolumeName, "1Gi", utils.InvalidQcowImagesURL)
+			By(fmt.Sprintf("creating new datavolume %s", dataVolume.Name))
+			dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolume)
+			Expect(err).ToNot(HaveOccurred())
+
+			//Due to the rate limit, this will take a while, so we can expect the phase to be in progress.
```

Comment:
I think you can remove the lines from 372 to 382. I am assuming you got most of this code from other tests, so here is some information:
We have 3 ports on the file host nginx container:
80 http, no rate limit
81 https, with self signed cert, you can get the certificate from the config map in the cdi namespace, there are existing functions that copy it into the testing namespace to be usable.
82 http, rate limited port. Same as port 80, but with a rate limit. This if for tests where we need it to take while for some resource in the import pod to become available (like the progress reporting). If the pod completes too fast we can't see if it is working.

In this case we are using port 80, so mention of the rate limit port in the comments confused me at first. But then I realized for this particular test (making sure the counter increases, we aren't actually interested in the phase of the data volume, so there is no need to make sure it is a certain value).

Long story short, you can likely remove the lines I mention above.